lightning/src/ln/chanmon_update_fail_tests.rs

```diff
@@ -29,10 +29,9 @@ use ln::functional_test_utils::*;
 
 use util::test_utils;
 
-#[test]
-fn test_simple_monitor_permanent_update_fail() {
+fn do_test_simple_monitor_permanent_update_fail(persister_fail: bool) {
```

Comment:
"If persister fail is true, we have the persister return a PermanentFailure instead of the higher-level ChainMonitor"